<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabstatistics</class>
 <widget class="QMainWindow" name="tabstatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEditBeginTime">
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="correctionMode">
           <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>至：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEditEndTime"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSum">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>   统计</string>
          </property>
          <property name="icon">
           <iconset resource="../res.qrc">
            <normaloff>:/image/image/tc.ico</normaloff>:/image/image/tc.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>取材总例数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>包埋总数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>常规取材例数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>常规包埋数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>冰冻取材例数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>冰冻包埋数</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReferesh"/>
   <addaction name="actionExtend"/>
  </widget>
  <action name="actionReferesh">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/image/refresh.png</normaloff>:/image/image/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="toolTip">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionExtend">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/image/excel.png</normaloff>:/image/image/excel.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>导出</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonSum</tabstop>
  <tabstop>dateTimeEditEndTime</tabstop>
  <tabstop>dateTimeEditBeginTime</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
